dvc/progress.py

```diff
@@ -1,11 +1,14 @@
 """Manages progress bars for dvc repo."""
 from __future__ import print_function
 import logging
+import sys
+import os
 from tqdm import tqdm
 from concurrent.futures import ThreadPoolExecutor
 from funcy import merge
 
 logger = logging.getLogger(__name__)
+TQDM_ISATTY = os.environ.get("TQDM_ISATTY", None)
```

Comment:
Feels weird to have `TQDM_` env var that is actually not parsed by tqdm. Maybe `DVC_ISATTY`? But then again, do we really need env var at all? The issue it is trying to solve is that on tests isatty will return False, so some code won't be tested. But that code is mostly about tqdm internals, right? So it is already tested by tqdm tests. So maybe we could just mock isatty where we actually care about tqdm output? @casperdcl @Suor What do you think, guys?

Reply:
I think @Suor wanted and env var more because of redirecting dvc output to a file and wanting to force progressbars to still appear for debugging purposes.

Reply:
@casperdcl Ah, right. We've had a discussion about it today with @Suor , i see his point when talking about benchmarking so I'm fine with having that env var. Ok, let's rename it at least then. And please see https://github.com/iterative/dvc/pull/2550#discussion_r334470806

Reply:
any suggestions? `DVC_PROGRESS_TTY`?

Reply:
@casperdcl DVC_ISATTY is fine